### PR TITLE
Fixed game crashing when attempting to load a previous menu.

### PR DESCRIPTION
**Pull request by Dragon20C**
Since we are not deferring anymore, needed to replace deferred_menu_load to load_menu.

### CODE REVIEW

File: scripts/menu_system/menu_system.gd [hunk 1]
```diff
@@ -66,7 +66,7 @@ func load_menu(key : Keys) -> void:
 func load_previous_menu() -> void:
 	## Delete the last menu in the previous menu array and load it
 	previous_menus.pop_back()
-	call_deferred("deferred_menu_load",previous_menus[-1])
+	call_deferred("load_menu",previous_menus[-1])
```
Comment (ChaoticTechSupport) [inline scripts/menu_system/menu_system.gd:69]: does it still need to be deferred?

**Review by ChaoticTechSupport — CHANGES_REQUESTED** [review]
hi, quick question.

File: scripts/menu_system/menu_system.gd (hunk 1 above)
Comment (Dragon20C) [reply scripts/menu_system/menu_system.gd:69]: oh probably not.

**Review by ChaoticTechSupport — APPROVED** [review]
looks good!

**Pull request merged**